### PR TITLE
Scaffold Elements 

### PR DESCRIPTION
**Pull request by PoojaB26**
### Description
Provide a brief overview of what this documentation update is about. Explain what sections or topics are being added or revised.

Linear ticket and [magic word](https://linear.app/docs/github#link-prs) Fixes DEVR-375

## Type of change
- [ ] Typo fix 
- [x] New feature 
- [ ] Removed outdated references

### CODE REVIEW

File: docs/resources/ui-building-blocks/pages/page-elements.md [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+slug: scaffold-elements
+sidebar_position: 2
+toc_max_heading_level: 5
+---
+
+# Scaffold Elements
+
+:::info
+To learn about Scaffold and the various page elements under it, refer to the [Behind a
+FlutterFlow Page](pages.md#behind-a-flutterflow-page) section.
```
Comment (pinkeshmars) [inline docs/resources/ui-building-blocks/pages/page-elements.md:12]: 1. The link path is not found
2. [**Behind a FlutterFlow Page**] bold

File: docs/resources/ui-building-blocks/pages/page-elements.md [hunk 2]
```diff
@@ -0,0 +1,414 @@
...
+- **Leading:** Typically holds a menu or back button, providing navigation control.
+- **Title:** Primarily serves to indicate the content of the active screen or to display the
+  name of the application, aiding users in recognizing their context within the app. This section
+  can also be customized with different widgets for a more tailored visual representation.
+- **Actions:** Hosts icon buttons for various operations like search, share, and more, situated on
+  the right end.
+
+### Add an AppBar to your Page
+
+<div style={{
```
Comment (pinkeshmars) [inline docs/resources/ui-building-blocks/pages/page-elements.md:32]: 1. Add period '.' in some of the callout text.
2. Typo in 6th slide. it should be 'Widget Palette'.

File: docs/resources/ui-building-blocks/pages/page-elements.md [hunk 3]
```diff
@@ -0,0 +1,414 @@
...
+    </iframe>
+</div>
+
+## Drawers
+
+**Drawer** is a slide-out menu that can emerge from either side of the screen, typically used for
+app navigation or placing additional options. It allows users to switch between different sections
+of an app without cluttering the main interface.
+
+### Add a Drawer to your Page
```
Comment (pinkeshmars) [inline docs/resources/ui-building-blocks/pages/page-elements.md:112]: In the Arcade, maybe we can have a third slide that shows the hotspot on the drawer tile. Just like the third slide in the FAB arcade.

File: docs/resources/ui-building-blocks/pages/page-elements.md [hunk 4]
```diff
@@ -0,0 +1,414 @@
...
+By default, the NavBar is disabled for any project created in FlutterFlow.
+Before you can add pages to the NavBar, you need to enable it from the
+FlutterFlow settings. Navigate to **Setting and Integrations > General > NavBar & AppBar** and
+enable Nav Bar.
+
+:::caution
+Initially, your NavBar will not have any pages in it. You'll see a message
+instructing you to add at least two pages. Before proceeding, make sure to
+create at least two pages. If you need help with adding a new page, you can find
+[more information here](how-tos.md#create-an-empty-page).
```
Comment (pinkeshmars) [inline docs/resources/ui-building-blocks/pages/page-elements.md:170]: 1. link path leading to page not found
2. [**more information here**] bold

File: docs/resources/ui-building-blocks/pages/page-elements.md [hunk 5]
```diff
@@ -0,0 +1,414 @@
...
+- **Unselected Icon & Text Color:** Defines the color for the icons and text when they are not
+  selected.
+- **Selected Background Color:** Alters the background color of the selected item.
+- **Width:** Controls the width of the NavBar.
+- **Border Radius:** Determines the roundness of the NavBar's corners.
+- **Elevation:** Adjusts the shadow or elevation effect beneath the NavBar, which helps give
+  the NavBar a floating appearance above other content.
+- **Button Border Radius:** Specifies the radius for the borders of each button within the NavBar.
+- **Nav Button Margin:** Sets the margin around each nav button
+- **Nav Button Padding:** Controls the padding inside each nav button.
```
Comment (pinkeshmars) [inline docs/resources/ui-building-blocks/pages/page-elements.md:305]: Maybe we could add [this](https://youtu.be/Qhe8X5ykK54) youtube video after this?

**Review by pinkeshmars — CHANGES_REQUESTED** [review]

File: docs/resources/ui-building-blocks/pages/page-elements.md (hunk 1 above)
Comment (PoojaB26) [reply docs/resources/ui-building-blocks/pages/page-elements.md:12]: changed to bold, but page isnt written yet

File: docs/resources/ui-building-blocks/pages/page-elements.md (hunk 2 above)
Comment (PoojaB26) [reply docs/resources/ui-building-blocks/pages/page-elements.md:32]: Done

File: docs/resources/ui-building-blocks/pages/page-elements.md (hunk 5 above)
Comment (PoojaB26) [reply docs/resources/ui-building-blocks/pages/page-elements.md:305]: Great catch! Added

File: docs/resources/ui-building-blocks/pages/page-elements.md (hunk 3 above)
Comment (PoojaB26) [reply docs/resources/ui-building-blocks/pages/page-elements.md:112]: Fixed!

**Comment by PoojaB26** [comment]
@pinkeshmars Addressed comments, please check and approve if everything is sorted.

**Review by pinkeshmars — APPROVED** [review]
@PoojaB26 LGTM. Thanks for working on these.

**Pull request merged**